worlds/sm64ex/Rules.py

```diff
@@ -191,14 +195,14 @@ def set_rules(world, options: SM64Options, player: int, area_connections: dict,
     rf.assign_rule("Vanish Cap Under the Moat Switch", "WK/TJ/BF/SF/LG | MOVELESS")
     rf.assign_rule("Vanish Cap Under the Moat Red Coins", "TJ/BF/SF/LG/WK & VC | CAPLESS & WK")
     # Bowser in the Fire Sea
-    rf.assign_rule("BitFS: Upper", "CL")
+    rf.assign_rule("BitFS: Upper", "CL | MOVELESS & TJ+WK")
     rf.assign_rule("Bowser in the Fire Sea Red Coins", "LG/WK")
```

Comment:
Can we add `| MOVELESS` to BitFS Red Coins and 1Up Block?  It is possible to get up top by landing on the thin wall from the pole, and it's still possible with WK.

Reply:
I think that's fair, yup. Added.

Reply:
Thanks.  Will continue to test around with the logic.